test/models/upvote_test.rb

```diff
@@ -1,7 +1,22 @@
 require 'test_helper'
 
 class UpvoteTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+
+	test "Comment ID uniqueness" do 
+		upvote = Upvote.new(:user_id => 123123 , :comment_id => 456456)
```

Comment:
I realized I only needed to check for comment ID's uniqueness and the user_id was only to test for presence. I made changes the "comment_id uniqueness" test I will push. please let me know if I understood what you meant.